internal/dataplane/parser/translate_httproute.go

```diff
@@ -26,25 +26,16 @@ func (p *Parser) ingressRulesFromHTTPRoutes() ingressRules {
 		return result
 	}
 
-	var errs []error
 	for _, httproute := range httpRouteList {
 		if err := p.ingressRulesFromHTTPRoute(&result, httproute); err != nil {
-			errs = append(errs, fmt.Errorf("HTTPRoute %s/%s can't be routed: %w",
-				httproute.Namespace, httproute.Name, err),
-			)
+			p.registerTranslationFailure(fmt.Sprintf("HTTPRoute can't be routed: %s", err), httproute)
```

Comment:
Why change `%w` to `%s` to print error here? I think `%w` is the recommended way to format errors.